examples/encfs/aci-arm-template.json

```diff
@@ -36,7 +36,7 @@
                   "value": "<base64-std-encodes-string-of-sidecar-args>"
                 }
               ],
-              "image": "<registry-name>/encfs:<tag_version>",
+              "image": "mcr.microsoft.com/aci/encfs:main_20230216.1",

```

Comment:
Who built this container and what commit id was used to build it?

Reply:
Commit Id = a82b530 